README.md

```diff
@@ -19,6 +19,13 @@ Node FHIR client library
 
 [JSDoc-generated documentation](https://vermonster.github.io/node-fhir-client/fhir-client/0.1.0/)
 
+# Logging
+
+The [debug library](https://www.npmjs.com/package/debug) can provide logging during development. Two different logging namespaces are provided, `node-fhir-client:info` logs each request and response, and `node-fhir-client:error` logs errors. To enable logging during development, add one of the namespaces to the DEBUG environment variable, or use `node-fhir-client:*` to enable both.
```

Comment:
Set to 80 wide. In vim, make selection and type `gq` to reformat. See: https://github.com/google/styleguide/blob/gh-pages/docguide/style.md